mqtt/mqtt-edgehub/tests/command.rs

```diff
@@ -28,13 +27,13 @@ async fn disconnect_client() {
 
     let server_handle = start_server(broker, DummyAuthenticator::anonymous());
 
-    let (mut command_handler_shutdown_handle, join_handle) =
+    let (command_handler_shutdown_handle, join_handle) =
         start_command_handler(broker_handle, TEST_SERVER_ADDRESS.to_string())
             .await
             .expect("could not start command handler");
 
     // TODO: This wait is necessary because edgehub can send disconnect before command handler subscribes to disconnect topic
-    //       We can remove once we have a proper approach for starting command handler before edgehub sends disconnects
+    //       We will soon have a proper approach for starting command handler before edgehub sends disconnects
```

Comment:
Once we have this proper startup logic, we still can't remove this wait unless we make other changes to the test. I'm not sure how this is going to look.

Reply:
I thought that by the time `start_command_handler` completes, the CH already subscribed? so there is no reason for `defay_for` anymore.

Reply:
Oh you are correct.